### DISCUSSION TITLE
Thank You! for creating this

### DISCUSSION THREAD

**Discussion by SavaMihai** [Didstopia/valheim-server#4] [General]
Thank You @Dids for creating and supporting this project.
I forked your repo and built the image myself, everything is working perfectly.

I'm working now at a Terraform script that it's creating an Azure Container Instance in Azure for server hosting, it's cheap and reliable :) . It's already up and running the cost estimate it's around 5-10 $ / month for it but will see how it goes.

**Comment by ping38** [comment]
Thanks @Dids for this cool Docker container ... "always beware of Vikings coming from Finland"
